lib/bundle/utils.rb

```diff
@@ -16,16 +16,10 @@ def self.system(cmd, *args)
     success
   end
 
-  def self.brew_installed?
-    @brew ||= Kernel.system("brew --version >/dev/null 2>&1")
-  end
-
   def self.cask_installed?
-    @cask ||= Kernel.system("brew command cask >/dev/null 2>&1")
-  end
-
-  def self.services_installed?
-    @services ||= Kernel.system("brew command services >/dev/null 2>&1")
+    @cask ||= begin
+      which("brew-cask") || which("brew-cask.rb")
```

Comment:
More performant to not shell out and I figured this was nicer than `require "cmd/command"`